src/gei/Commands/MigrateRepoCommand.cs

```diff
@@ -100,6 +102,14 @@ public class MigrateRepoCommand : CommandBase<MigrateRepoCommandArgs, MigrateRep
         {
             Description = "If uploading to S3, the AWS secret key. If not provided, it will be read from AWS_SECRET_KEY environment variable. Not required if migrating from GitHub Enterprise Server 3.8.0 or later."
         };
+        public Option<string> AwsSessionToken { get; } = new("--aws-session-token")
+        {
+            Description = "If using AWS, the AWS session token. If not provided, it will be read from AWS_SESSION_TOKEN environment variable."
+        };
+        public Option<string> AwsRegion { get; } = new("--aws-region")
+        {
+            Description = "If using AWS, the AWS region. If not provided, it will be read from AWS_REGION environment variable. Defaults to us-east-1."
```

Comment:
```suggestion
            Description = "If using AWS, the AWS region. If not provided, it will be read from AWS_REGION environment variable. Defaults to us-east-1 if neither the argument nor the environment variable is set. In a future release, you will be required to set an AWS region if using AWS S3 as your blob storage provider."
```

Reply:
Addressed in 98de164b93e6c5ad0a9f24c52673cd80b4f9a7fc